developer_README.md

```diff
@@ -89,36 +57,15 @@ Nothing else.
 
 Note that many App features will be set and updated with information from the `appspec.json` in each new App version. Thus, even if not bundled into the App, this file is required and must be up to date.
 
+You are also required to write a documentation file (see [Step 6](https://docs.moveapps.org/#/create_app#step-6-write-a-documentation-file) in the User Manual).
 
-## App development
-
-1. Execute `Rscript sdk.R` (on a terminal) or run/source `sdk.R` in _RStudio_ (_Run App_)
-1. Ensure the sdk executes the vanilla template App code. Everything is set up correctly if no error occurs and you can open the Shiny App in your local Web-Browser.
-1. Begin with your App development in `./ShinyModule.R`
-
-## Examples
-
-### Request App configuration from your users
-
-tbd (Shiny Bookmark instructions)
-
-### Produce an App artefact
-
-tbd (should be supported by SDK - must be tested by sample app implementation. Provide code snippets)
-
-## Include files to your App
-
-[Details and examples about _auxiliary files_](https://docs.moveapps.org/#/auxiliary).
-
-This template also implements in `./ShinyModule.R` a showcase about this topics.
-
----
 
 ## R packages management / renv (optional)
 
 The template is prepared to use [`renv` as a dependency manager](https://rstudio.github.io/renv/articles/renv.html) - but is disabled by default (_opt-in_).
 You can [activate `renv` with `renv::activate()`](https://rstudio.github.io/renv/articles/renv.html#uninstalling-renv) and then use it in the [usual `renv` workflow](https://rstudio.github.io/renv/articles/renv.html#workflow).
 
+
 ### Docker support (optional)
 
 - at the end your app will be executed on MoveApps in a Docker container.
```

Comment:
```suggestion
Your app will be executed on MoveApps in a Docker container. Specially for debugging errors that are not straight forward, it can be very useful to execute your App in a docker environment. These more complex errors are often due to system libraries that need to be installed in MoveApps to run the App. The easiest way to find out which ones they are is to run the App locally in a docker environment:
```